htdocs/PHP/internal_libraries/languages/page_content_fr.php

```diff
@@ -93,6 +93,7 @@
 $translation["done_page_content"] = "
 <p class='text-center'>Votre note d'urgence a été créée. Vous pouvez le télécharger en cliquant sur le bouton ci-dessous, puis l'imprimer et la remettre à votre personne de confiance.</p>
 <p class='text-center'><b><mark>Très important !</mark></b> Ajoutez l'adresse e-mail <b>support@weexpire.org</b> à la liste des expéditeurs sûrs de votre fournisseur de mails pour éviter que toute notification de notre part concernant la note que vous venez de créer ne se retrouve par erreur dans les spams.</p>";
+$translation["done_page_donate_message"] = "<p class='text-center'><small>Soutenez WeExpire avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.</small></p>";
```

Comment:
@eze-kiel I think it's correct in French but it's always better to double check it :)

Reply:
:+1: